tests/testlib/s2n_testlib.h

```diff
@@ -130,6 +130,12 @@ int s2n_connection_set_all_protocol_versions(struct s2n_connection *conn, uint8_
 #define S2N_ONE_TRAILING_BYTE_CERT_BIN         "../pems/one_trailing_byte_cert.bin"
 #define S2N_TWO_TRAILING_BYTE_CERT_BIN         "../pems/two_trailing_byte_cert.bin"
 
+/* This is a certificate chain with a SHA-1 legacy algorithm on the root and 
+ * a SHA-256 signature algorithm on the intermediate certificate. This is used to prove
+ * that our certificate validation code does not fail a root certificate signed with SHA-1.
+ */
+#define S2N_SHA1_ROOT_SIGNATURE_CERT_CHAIN     "../pems/rsa_sha1_root_signature_chain.pem"
```

Comment:
We could follow the naming conventions in ../pems/ folder and name the file as "rsa_1024/2048/4096_sha1_root_signature_chain.pem"